runtime/vm/UpcallThunkMem.cpp

```diff
@@ -87,66 +87,100 @@ subAllocateThunkFromHeap(J9UpcallMetaData *data)
 {
 	J9JavaVM * vm = data->vm;
 	UDATA thunkSize = data->thunkSize;
+	J9UpcallThunkHeapList *thunkHeapHead = vm->thunkHeapHead;
+	J9UpcallThunkHeapList *thunkHeapNode = NULL;
+	J9UpcallThunkHeapWrapper *thunkHeapWrapper = NULL;
 	J9Heap *thunkHeap = NULL;
```

Comment:
These two locals can move into the loop below.

Reply:
Agreed and moved them into the loop.